WordPress/src/main/AndroidManifest.xml

```diff
@@ -21,7 +21,8 @@
     <!-- Required for storing and retrieving screenshots, taking photos, accessing media files -->
     <uses-permission
         android:name="android.permission.WRITE_EXTERNAL_STORAGE"
-        android:maxSdkVersion="29" />
+        android:maxSdkVersion="29"
+        tools:replace="android:maxSdkVersion" />
```

Comment:
This is already mentioned in the description, but I'm pasting it here as well to make sure it's not missed.

---

You can see that I had to add `tools:replace="android:maxSdkVersion"` to the `AndroidManifest.xml`. This ensures that no changes to the WP manifest are done and the `maxSdkVersion=29` is kept. Quick Editor needs the `android.permission.WRITE_EXTERNAL_STORAGE` permission but only up to API 28, so that's where the conflict is. This permission is needed for the DownloadManager (see [docs](https://developer.android.com/reference/android/app/DownloadManager.Request#setDestinationInExternalPublicDir(java.lang.String,%20java.lang.String))) to download Avatars. 

Here's a screenshot of the merge AndroidManifest from the .apk confirming no changes are made. 

<img width="519" alt="Screenshot 2024-12-18 at 11 53 40" src="https://github.com/user-attachments/assets/8def4035-d60e-474b-b237-99235b5e2074" />